addons/pos_sale_loyalty/static/src/tours/PosSaleLoyaltyTour.js

```diff
@@ -0,0 +1,21 @@
+/** @odoo-module **/
+
+import { Chrome } from 'point_of_sale.tour.ChromeTourMethods';
+import { PaymentScreen } from 'point_of_sale.tour.PaymentScreenTourMethods';
+import { ProductScreen } from 'pos_sale.tour.ProductScreenTourMethods';
+import { getSteps, startSteps } from 'point_of_sale.tour.utils';
+import Tour from 'web_tour.tour';
+
+// First tour should not get any automatic rewards
+startSteps();
+
+ProductScreen.do.confirmOpeningPopup();
+ProductScreen.do.clickQuotationButton();
+ProductScreen.do.selectFirstOrder();
+ProductScreen.do.clickDisplayedProduct('Desk Pad');
+ProductScreen.do.clickPayButton();
+PaymentScreen.do.clickPaymentMethod('Bank');
+PaymentScreen.do.clickValidate();
+Chrome.do.clickTicketButton();
```

Comment:
We can merge but change this last step. it should to check if the screen is receipt screen.